app/models/job.rb

```diff
@@ -13,4 +13,12 @@ def assign_tech
     tech_matches = Technology.where(name: raw_technologies)
     self.technologies = tech_matches
   end
+
+  def company_name
```

Comment:
Thanks for these links! I really like the way decorators handle presentation so I'll work on that next :+1: